platform/nephos/sdk.mk

```diff
@@ -1,4 +1,4 @@
-NEPHOS_NPS_KERNEL = nps-modules-3.16.0-5_2.0.3_3147dc_amd64.deb
-$(NEPHOS_NPS_KERNEL)_URL = "https://github.com/NephosInc/SONiC/raw/master/sdk/nps-modules-3.16.0-5_2.0.3_3147dc_amd64.deb"
+NEPHOS_NPS_KERNEL = nps-modules-4.9.0-7_2.0.5_fc2e9a_amd64.deb
+$(NEPHOS_NPS_KERNEL)_URL = "https://github.com/NephosInc/SONiC/raw/master/sdk/nps-modules-4.9.0-7_2.0.5_fc2e9a_amd64.deb"
```

Comment:
kernel module needs to be GPL licensed.

looks like none of you kernel modules are.
```
lgh@gulv-vm1:~/hehe$ modinfo ./lib/modules/4.9.0-7-amd64/extra/nps_dev.ko
filename:       /home/lgh/hehe/./lib/modules/4.9.0-7-amd64/extra/nps_dev.ko
description:    SDK Kernel Module
author:         Nephos
license:        Nephos
depends:        
retpoline:      Y
vermagic:       4.9.0-7-amd64 SMP mod_unload modversions 
lgh@gulv-vm1:~/hehe$ modinfo ./lib/modules/4.9.0-7-amd64/extra/nps_netif.ko
filename:       /home/lgh/hehe/./lib/modules/4.9.0-7-amd64/extra/nps_netif.ko
description:    NETIF Kernel Module
author:         Nephos
license:        Nephos
depends:        nps_dev
retpoline:      Y
vermagic:       4.9.0-7-amd64 SMP mod_unload modversions 
parm:           dbg_flag:bit0:Error, bit1:Tx, bit2:Rx, bit3:Intf, bit4:Profile (uint)
```